### ISSUE TITLE
"deep link" from Python/R client session to web editor

### ISSUE THREAD

**Issue by soxofaan** [Open-EO/openeo-web-editor#262]
Feature idea (to be discussed how useful/feasible this is):
- user is (for example) using python client in a jupyter environment and has authenticated connection
- user does something like `connection.open_webeditor()`
- new browser window opens (or some kind of link is printed)  with authenticated webeditor, so user does not have to go through authentication on webeditor again

**Comment by soxofaan** [comment]
(hard part of this is probably getting the auth tokens from python space to browser space is a secure manner)

**Comment by m-mohr** [comment]
Yes, the authentication issue is really the hardest part, I think.

And there's the rabbit hole: Which editor to open? editor.openeo.cloud or editor.openeo.org ;-)

**Comment by soxofaan** [comment]
> Which editor to open? editor.openeo.cloud or editor.openeo.org ;-)

That could be done with an argument e.g. `connection.open_webeditor("editor.openeo.org")`, a client config  file entry or ultimately even a discovery/detection mechanism (e.g. something along the lines of prepending "editor." to the back-end hostname)

**Comment by soxofaan** [comment]
this idea of forwarding the token obtained with python client to web editor could also help with
- https://github.com/Open-EO/openeo-web-editor/issues/324
